source/dsl/GitDSL.ts

```diff
@@ -64,8 +76,7 @@ export interface GitDSL {
    * @param {string} filename the path to the json file
    * @param {string[]} diffTypes An array containing: "add", "del" or "normal" - then you can only get these changes
    */
-
-  diffForFile(filename: string, diffTypes?: string[]): string | null,
+  diffForFile(filename: string, diffTypes?: string[]): Promise<TextDiff | null>,
```

Comment:
Should we still keep the `added`/`removed`/`normal` changes? They're not really the same as `JSONDiff`.